### PR TITLE
Planning: bug fix complete for iterative anchoring smoother

### PR DESCRIPTION
**Pull request by JasonZhou404**

### CODE REVIEW

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc [hunk 1]
```diff
@@ -94,8 +94,10 @@ bool IterativeAnchoringSmoother::Smooth(
   DiscretizedPath interpolated_warm_start_path;
   // TODO(Jinyun): move to confs
   const double interpolated_delta_s = 0.1;
-  const double path_length = warm_start_path.Length();
-  for (double s = 0; s < path_length; s += interpolated_delta_s) {
+  double path_length = warm_start_path.Length();
+  double delta_s = path_length / std::ceil(path_length / interpolated_delta_s);
```
Comment (Capri2014) [inline modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: @JasonZhou404 Need to protect against dividing by zero here.

Comment (JasonZhou404) [reply modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: Not likely the two denominators would be zero, but is indeed a valid concern if the interpolated_delta_s by config is set as zero. Will fix it in future commits

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc [hunk 2]
```diff
@@ -192,10 +145,17 @@ bool IterativeAnchoringSmoother::Smooth(
...
+  // if (!GenerateStopProfileFromPolynomial(
+  //         init_a, init_v, smoothed_path_points.Length(), &smoothed_speeds)) {
+  //   return false;
+  // }
+
   const auto speed_smooth_end_timestamp = std::chrono::system_clock::now();
   std::chrono::duration<double> speed_smooth_diff =
       speed_smooth_end_timestamp - speed_smooth_start_timestamp;
-  ADEBUG << "iterative anchoring speed smoother time: "
+  AERROR << "iterative anchoring speed smoother time: "
```
Comment (ycool) [inline modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: s/AERROR/ADEBUG/

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc [hunk 3]
```diff
@@ -225,13 +258,13 @@ bool IterativeAnchoringSmoother::SmoothPath(
 
   FemPosDeviationSmoother fem_pos_smoother;
   // TODO(Jinyun): move to confs
-  fem_pos_smoother.set_weight_fem_pos_deviation(1e5);
+  fem_pos_smoother.set_weight_fem_pos_deviation(1.0e8);
```
Comment (Capri2014) [inline modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: Better reformulate the problem for dimension normalization in next iteration, to increase robustness and make future tuning easier.

File: modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc (hunk 2 above)
Comment (JasonZhou404) [reply modules/planning/open_space/trajectory_smoother/iterative_anchoring_smoother.cc]: fixed

**Pull request merged**